modules/cpmso-yaml-failure-domain-aws.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-failure-domain-aws_{context}"]
+= Sample AWS failure domain configuration
+
+The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible.
+
+.Sample AWS failure domain values
+[source,yaml]
+----
+failureDomains:
+  aws:
+  - placement:
+      availabilityZone: <aws_region_1> <a>
+    subnet:
+      filters:
+      - name: tag:Name <b>
+        values:
+        - <cluster_id>-private-<aws_region_1> <c>
+      type: Filters
+  - placement:
+      availabilityZone: <aws_region_2> <d>
+    subnet:
+      filters:
+      - name: tag:Name <b>
+        values:
+        - <cluster_id>-private-<aws_region_2> <e>
+      type: Filters
+  platform: AWS <f>
+----
+<a> Specifies an AWS region for the first failure domain.
+
+<b> #Does this need any information?#
```

Comment:
Do we have any other examples of how to configure the subnets in the docs?

Reply:
not that I can find for "normal" cases - I am seeing more detail in VPC content, but the general assumption is that the installer does this.